test/e2e/operator/nmstate_install_test.go

```diff
@@ -89,11 +90,18 @@ var _ = Describe("NMState operator", func() {
 			installOperator("nmstate")
 		})
 		It("should wait on the old one to be deleted", func() {
+			newDaemonSetKey := types.NamespacedName{Namespace: operatorNamespace, Name: "nmstate-handler"}
```

Comment:
Can we wait in `installOperator` function?

Reply:
Is not the reight place since you can be installing the operator but not the NMState CR so there is no handler to check, in this case we have to check the handler because the NMState CR is already there from previous install so operator start the deploy but from different namespace.

Reply:
Got it, thanks